ParisBNF/abb/BNFabb147.xml

```diff
@@ -38,10 +38,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                              <material key="parchment"/>
                           </support>
                           <extent>
-                            <measure unit="page">i+128</measure>
-                            <note> The first guard leaf is designated “A” in the original pagination,
-                              in the description below its two pages are assigned numbers “i” and “ii”.
-                     </note>
+                            <measure unit="page">136 (2+128+2+2+2)</measure>
+                             <measure unit="quire">10</measure>
+                            <note>The first guard leaf appears to have been designated “A” in the original pagination,
+                             here its two pages are assigned numbers “i” and “ii”. Three pieces of paper (<ref target="p2"/>,
+                             <ref target="p3"/>, <ref target="p4"/>) seem to have been accomodated between <locus target="#28">p. 28</locus>
+                           and <locus target="#29">p. 29</locus>. Below they are assigned page numbers 129-130, 131-132, and 133-134, respectively.</note>
```

Comment:
page 28 corresponds to fol. 14v, so  it must be locus target="#14v" 
page 29 corresponds to fol. 15r, so it must be locus target="#15r" 
etc.
(we only write folia in locus!)

Reply:
also below, throughout the manuscript.

Reply:
you do not need to write "p.", this will be printed if extent="page"

you should write `<locus target="#15r">29</locus>`

Reply:
ok